docs/guides/vnc_selenium.rst

```diff
@@ -85,7 +85,7 @@ Here's an example script that does those things:
 Important things:
 * The script **MUST** start with `#!/bin/sh` (or your shell shebang of choice).
 * The script **MUST** be executable (`chmod +x ~/.vnc/xstartup`)
-
+* The "-ensureCleanSession -trustAllSSLCertificates"  won't work with the latest selenium-server which is 3.4.0. 
```

Comment:
I don't think defining selenium server version is needed in such context. I'd write "The '-ensureCleanSession' and '-trustAllSSLCertificates' keys have removed since 3.x version of selenium server".